### PR TITLE
Some Change in Gui

### PR DESCRIPTION
**Pull request by Zangetsu38**
See here for a list of changes made. https://github.com/RPCS3/rpcs3/issues/1312
Change Key for Resume with "F", Key "E" is already use for R2, so In actualy conflict With R2

### CODE REVIEW

File: rpcs3/Gui/AutoPauseManager.cpp [hunk 1]
```diff
@@ -23,7 +23,7 @@ AutoPauseManagerDialog::AutoPauseManagerDialog(wxWindow* parent)
 
 	m_entry_convert.clear();
 	m_entry_convert.str("");
-	m_entry_convert << "To use auto pause: enter the ID(s) of a function or a system call. Restart of the game is required to apply. You can enable/disable this in the settings.";
+	m_entry_convert << "To use auto pause: enter the ID(s) of a function or a system call. You mus restart your emulated game or application for changed settings to take effect. You can enable/disable this in the settings.";
```
Comment (paulsapps) [inline rpcs3/Gui/AutoPauseManager.cpp]: mus -> must

Comment (Zangetsu38) [inline rpcs3/Gui/AutoPauseManager.cpp]: oups :3

Comment (Zangetsu38) [inline rpcs3/Gui/AutoPauseManager.cpp]: Coorected

File: rpcs3/Gui/GameViewer.h [hunk 2]
```diff
@@ -88,12 +88,12 @@ struct ColumnsArr
...
 
 		m_columns.clear();
-		m_columns.emplace_back(m_columns.size(),  90, "Icon");
+		m_columns.emplace_back(m_columns.size(),  90, "");
 		m_columns.emplace_back(m_columns.size(), 160, "Name");
 		m_columns.emplace_back(m_columns.size(),  85, "Serial");
-		m_columns.emplace_back(m_columns.size(),  55, "FW");
-		m_columns.emplace_back(m_columns.size(),  55, "App version");
-		m_columns.emplace_back(m_columns.size(),  75, "Category");
+		m_columns.emplace_back(m_columns.size(),  55, "Min fw");
```
Comment (tambry) [inline rpcs3/Gui/GameViewer.h]: Nitpicking here, but could you put a dot here, after the "Min", since it's a shortened word. Like so: "Min. FW"
Same for other shortened words, would be nice.

Comment (Zangetsu38) [inline rpcs3/Gui/GameViewer.h]: Updated

Comment (paulsapps) [inline rpcs3/Gui/GameViewer.h]: Extra nitpicking, why is one "FW" and the other is "fw"? Why not "Minimum firmware version"?

Comment (tambry) [inline rpcs3/Gui/GameViewer.h]: @paulsapps Which other? This is an old diff you're commenting on. And the reason for that would most likely be, that the row would take up most of the space in the game viewer.

**Comment by Zangetsu38** [comment]
@tambry It's all good like this?

File: rpcs3/Gui/LLEModulesManager.cpp [hunk 3]
```diff
@@ -8,8 +8,9 @@
 #include "Emu/state.h"
 #include "Emu/FS/VFS.h"
 
-LLEModulesManagerFrame::LLEModulesManagerFrame(wxWindow* parent) : wxFrame(parent, wxID_ANY, "LLEModulesManagerFrame")
+LLEModulesManagerFrame::LLEModulesManagerFrame(wxWindow* parent) : wxFrame(parent, wxID_ANY, "LLE modules manager frame")
```
Comment (tambry) [inline rpcs3/Gui/LLEModulesManager.cpp]: Would be nice to remove the "frame" at the end of the name. There's no need to indicate to an user, that it's a window frame.

Comment (Zangetsu38) [inline rpcs3/Gui/LLEModulesManager.cpp]: ok, is done

**Comment by tambry** [comment]
@Zangetsu38 Needs a rebase

**Comment by Zangetsu38** [comment]
@tambry Rebased

**Comment by Nekotekina** [comment]
Remove all MsgDialog or icon changes, they were already rejected.

**Comment by Zangetsu38** [comment]
@Nekotekina Why remove ?
It is Just Logic Here !
For Icon, I have add only in Windows Requiered
Exemple : 
capture d ecran 2015-11-26 00 34 37
No Icon here, Very Strange
Now with this Commit :
capture d ecran 2015-11-26 00 36 25
Now it is a normaly.

After for MsgDialog: 
Why this it is a whrite "Error" Totally Illogic, it is not error, but only information, All Windows this show me for me it is a information of installation.
And with this transparency just very hard read "Information"
capture d ecran 2015-11-26 00 40 29
With this Commit Now : 
capture d ecran 2015-11-26 00 51 38
More readable, and + clear for user.

**Comment by Nekotekina** [comment]
This doesn't prove anything. All "Atelier" games may share the same engine.

**Comment by Nekotekina** [comment]
About icons: they are wrongly implemented in my opinion. I'll look into it. You wanted to copypasta some code in as much windows as possible, which is not very good.

And anyway, you tried to bring these changes silently hoping that nobody notices.

**Comment by danilaml** [comment]
@Nekotekina what's the problem with these "separate" tools having their own window?

**Comment by O1L** [comment]
Frankly, these "improvements" do not make sense. Basically, it's just pointless renaming of the parameters.
And yes, it is all the more pointless in the light of the fact that we will soon get rid of wxWidgets (GUI will be rebuilt from scratch).
#include "frame_icon.xpm" in stdafx-gui.cpp - bad idea. It adds icons where they should not be.
Once again, I recommend you stop shitting in RPCS3 code.

**Comment by danilaml** [comment]
@O1L what's up with rpcs3 GUI revamp? Who works on it?

**Comment by O1L** [comment]
Not right now, but plans are available.
Here is an example of adequate solution of empty icons problem - https://github.com/Nekotekina/rpcs3/commit/a3b28af9baf3caad10af94badb3c291b334b9b03

**Comment by danilaml** [comment]
@O1L plans are nothing by themselves. There is no point in abandoning current GUI just because there are plans in some distant future to use something else.

**Comment by vlj** [comment]
@O1L I think somebody once told to switch to Qt but it was merely a wish rather than a plan.
The roadmap mentions to get rid of wxwidget dependency in emucore (which is necessary, it doesn't make sense to use wxWidget for things like xml parsing) but doesn't mention rewriting the GUI.

**Comment by Zangetsu38** [comment]
@O1L Excuse me but I do not shit in the code, as you said.
@danilaml Exactly, I just want fix currently GUI
@Nekotekina You deceive you, I see the other games stink on this window to appear, and it was never errors!
So it must be left as I did, it's just normal.
Each time it is for an installation or whatever, but it was never a mistake.

**Comment by Zangetsu38** [comment]
@Nekotekina I remove change MsgDialog

**Comment by Zangetsu38** [comment]
@Nekotekina So Now is good ?

**Comment by Zangetsu38** [comment]
@Nekotekina rebase and icon removed

**Comment by Zangetsu38** [comment]
rebased

**Comment by Zangetsu38** [comment]
@Nekotekina All Icon and MsgDialog removed and rebased

**Comment by Zangetsu38** [comment]
Rebased

**Comment by Zangetsu38** [comment]
Rebased

**Comment by Zangetsu38** [comment]
@Nekotekina Problem icon allays present for  Elf compiler and Cg diassm windows.
Exemple in Elf compiler :
capture d ecran 2015-12-05 13 34 59

**Comment by Zangetsu38** [comment]
rebased

**Comment by Zangetsu38** [comment]
Rebased

**Comment by Zangetsu38** [comment]
@Nekotekina So it's good now?

File: rpcs3/Gui/MainFrame.cpp [hunk 4]
```diff
@@ -567,7 +564,7 @@ void MainFrame::OnKeyDown(wxKeyEvent& event)
 	{
 		switch(event.GetKeyCode())
 		{
-		case 'E': case 'e': if(Emu.IsPaused()) Emu.Resume(); else if(Emu.IsReady()) Emu.Run(); return;
+		case 'F': case 'f': if(Emu.IsPaused()) Emu.Resume(); else if(Emu.IsReady()) Emu.Run(); return;
```
Comment (tambry) [inline rpcs3/Gui/MainFrame.cpp]: @Zangetsu38 Any reason for changing this?

File: rpcs3/Gui/SettingsDialog.cpp [hunk 5]
```diff
@@ -54,7 +54,7 @@ std::vector<std::string> GetAdapters()
 	}
 	else
 	{
-		LOG_ERROR(HLE, "Call to GetAdaptersInfo failed.");
+		LOG_ERROR(HLE, "Call to get adapters info failed.");
```
Comment (tambry) [inline rpcs3/Gui/SettingsDialog.cpp]: Would be nice to change it back, since GetAdaptersInfo is a function.

File: rpcs3/Gui/MainFrame.cpp (hunk 4 above)
Comment (Zangetsu38) [inline rpcs3/Gui/MainFrame.cpp]: @tambry Yes, look description

File: rpcs3/Gui/SettingsDialog.cpp (hunk 5 above)
Comment (Zangetsu38) [inline rpcs3/Gui/SettingsDialog.cpp]: @tambry ah ok, i change back

File: rpcs3/Gui/MainFrame.cpp (hunk 4 above)
Comment (tambry) [inline rpcs3/Gui/MainFrame.cpp]: I see.

**Comment by Zangetsu38** [comment]
@O1L Removed

**Comment by Zangetsu38** [comment]
Rebased

**Comment by Zangetsu38** [comment]
Rebased
@Nekotekina it's good now ?
If there is still a problem that remains, say, but normally it is the good.

**Comment by Zangetsu38** [comment]
hmm So... ?

**Comment by Zangetsu38** [comment]
@Nekotekina Now it is a good like this ?

**Comment by Zangetsu38** [comment]
@tambry @DHrpcs3  @Nekotekina Now it is a good like this ?

**Comment by Nekotekina** [comment]
@Zangetsu38 Shut up and wait.

**Comment by danilaml** [comment]
@Nekotekina watch your language. For what is he supposed to wait? What's left in this PR that raises question (except maybe config names change which can break inis, but that's unavoidable and better done sooner than latter)

**Comment by Zangetsu38** [comment]
@Nekotekina  Uh ok, but wait what?
this PR have already 25 days

**Comment by Nekotekina** [comment]
@danilaml *facepalm* Defending an idiot who deletes his message and writes another one?

**Comment by Zangetsu38** [comment]
@Nekotekina I just clear my message because I had rebase, that's all, and I did not say the same after.

**Comment by Nekotekina** [comment]
Anyway, I'm sceptical about this PR, about changing the case in most words; it contains some unnecessary changes.

Changing keys, seriously? 
pad
I can configure it in any way.

**Comment by Zangetsu38** [comment]
hu, just look issue https://github.com/RPCS3/rpcs3/issues/1312, 
i not change key, juste name little change

File: rpcs3/Gui/SettingsDialog.cpp [hunk 6]
```diff
@@ -91,6 +92,7 @@ std::vector<std::string> GetAdapters()
 SettingsDialog::SettingsDialog(wxWindow *parent, rpcs3::config_t* cfg)
 	: wxDialog(parent, wxID_ANY, "Settings", wxDefaultPosition)
 {
+	SetIcon(wxGetApp().m_MainFrame->GetIcon());
```
Comment (danilaml) [inline rpcs3/Gui/SettingsDialog.cpp]: No icon is needed here.

Comment (Zangetsu38) [inline rpcs3/Gui/SettingsDialog.cpp]: Sure ? i juste think is more godd look ^^
capture d ecran 2015-12-19 18 15 39

Comment (danilaml) [inline rpcs3/Gui/SettingsDialog.cpp]: Settings is a modal dialog so it doesn't need icon. That's general convention, at least on windows (look at other software like pcsx2 or dolphin), Anyway, let's not include "controversial" icon changes in this PR (for it to have any chance to be accepted).

Comment (Zangetsu38) [inline rpcs3/Gui/SettingsDialog.cpp]: @danilaml hmm i see, removed

File: rpcs3/Gui/CgDisasm.cpp [hunk 7]
```diff
@@ -10,13 +10,13 @@ BEGIN_EVENT_TABLE(CgDisasm, wxFrame)
 END_EVENT_TABLE()
 
 CgDisasm::CgDisasm(wxWindow* parent)
-	: wxFrame(parent, wxID_ANY, "Cg Disasm", wxDefaultPosition, wxSize(640, 480))
+	: wxFrame(parent, wxID_ANY, "Cg disassembler", wxDefaultPosition, wxSize(640, 480))
```
Comment (CalmCodeC) [inline rpcs3/Gui/CgDisasm.cpp]: why not uppercase "D" ?

File: rpcs3/config.h [hunk 8]
```diff
@@ -180,9 +183,6 @@ namespace convert
 			if (value == "59.94")
 				return rsx_frame_limit::_59_94;
 
-			if (value == "30")
-				return rsx_frame_limit::_30;
-
 			if (value == "60")
 				return rsx_frame_limit::_60;
```
Comment (CalmCodeC) [inline rpcs3/config.h]: hmm, why did you change the location of the conditional check ? also the location of  _30 in the enum class ?

**Comment by CalmCodeC** [comment]
one more thing: are these changes of Auto -> Automatic even necessary ? it just means more text o_0

File: rpcs3/config.h (hunk 8 above)
Comment (tambry) [inline rpcs3/config.h]: Probably, so they would be nicely ordered.

**Comment by paulsapps** [comment]
@CalmCodeC because if English isn't your native lang then it isn't always obvious what "Auto" is supposed to mean

**Comment by CalmCodeC** [comment]
I have few suggestions: change "Render" to "Renderer" ( instead of currently used "Rendering API")
change "D3D Adaptater" to "D3D Adapter" ( instead of currently awkward "DirectX 12 primary adapter")

**Comment by MyaniPT** [comment]
This could probably be closed, no?

**Comment by Nekotekina** [comment]
Probably.
